src/index.ts

```diff
@@ -43,4 +45,12 @@ app.addDatasource(restAPIDatasource);
 app.addDatasource(postgresDatasource);
 app.addDatasource(mysqlDatasource);
 
-console.log(JSON.stringify(app, null, 2));
+// console.log(JSON.stringify(app, null, 2));
+
+fs.writeFile(
+    `${app.getName()}.json`,
+    JSON.stringify(app, null, 2), function (err) {
+        if (err) {
+            return console.log(err);
+        }
```

Comment:
Lets add this in the app.create() function

Reply:
There's a toJson() function already in `App.ts`, do we add it to that?